packages/client/src/views/PrintCertificate/PDFUtils.ts

```diff
@@ -74,14 +75,14 @@ export function printCertificate(
   if (!userDetails) {
     throw new Error('No user details found')
   }
-  printPDF(
-    application.event === Event.BIRTH
-      ? offlineResource.templates.certificates.birth
-      : offlineResource.templates.certificates.death,
-    application,
-    userDetails,
-    offlineResource,
-    intl,
-    optionalData
-  )
+  // printPDF(
```

Comment:
Is this here by accident?

Reply:
Yes. I will remove this block of code. Thanks.